homeassistant/components/sensor/radarr.py

```diff
@@ -0,0 +1,246 @@
+"""
+Support for Radarr.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.radarr/
+"""
+import logging
+import time
+from datetime import datetime
+
+import requests
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import (CONF_API_KEY, CONF_HOST, CONF_PORT)
+from homeassistant.const import CONF_MONITORED_CONDITIONS
+from homeassistant.const import CONF_SSL
+from homeassistant.helpers.entity import Entity
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_DAYS = 'days'
+CONF_INCLUDED = 'include_paths'
+CONF_UNIT = 'unit'
+CONF_URLBASE = 'urlbase'
+
+DEFAULT_HOST = 'localhost'
+DEFAULT_PORT = 7878
+DEFAULT_URLBASE = ''
+DEFAULT_DAYS = '1'
+DEFAULT_UNIT = 'GB'
+
```

Comment:
Add `SCAN_INTERVAL`. I think that's enough if this sensor is updated every 5 or 10 min per default.